packages/server-wallet/src/utilities/workers/worker.ts

```diff
@@ -9,7 +9,9 @@ import {ServerWalletConfig} from '../../config';
 
 import {isStateChannelWorkerData} from './worker-data';
 
-const walletConfig = workerData as ServerWalletConfig;
+// We only expect a worker thread to use one postgres connection but we enforce it just to make sure
+const walletConfig = {...(workerData as ServerWalletConfig), postgresPoolSize: {min: 0, max: 1}};
```

Comment:
Can we utilize the type system to ensure this remains a valid manipulation of the wallet config?

This will probably need to be prettied up.
```suggestion
const walletConfig: ServerWalletConfig = {...(workerData as ServerWalletConfig), postgresPoolSize: {min: 0, max: 1}};
```